web-components/src/components/modal/CreateSellOrderModal.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { RegenModalProps } from '.';
+import {
+  CreateSellOrderForm,
+  CreateSellOrderProps,
+} from '../form/CreateSellOrderForm';
+import { FormModalTemplate } from './FormModalTemplate';
+
+interface CreateSellOrderModalProps
+  extends RegenModalProps,
+    CreateSellOrderProps {
+  title: string;
+}
+
+const CreateSellOrderModal: React.FC<CreateSellOrderModalProps> = ({
```

Comment:
I've created both the Modal and the Form in web-components as all the others for MyEcoCredits page are also located here. That being said I have the feeling that they might fit better directly in registry.
Happy to create the t ask/issue to refactor/move them all at once if needed.